asm/Game/zMusic.s

```diff
@@ -7,13 +7,13 @@
 volume_reset__Fv:
 /* 800A6E9C 000A3C9C  94 21 FF F0 */	stwu r1, -0x10(r1)
 /* 800A6EA0 000A3CA0  7C 08 02 A6 */	mflr r0
-/* 800A6EA4 000A3CA4  3C 60 80 2F */	lis r3, lbl_802F2B38@ha
+/* 800A6EA4 000A3CA4  3C 60 80 2F */	lis r3, volume@ha
 /* 800A6EA8 000A3CA8  38 80 00 00 */	li r4, 0
 /* 800A6EAC 000A3CAC  90 01 00 14 */	stw r0, 0x14(r1)
-/* 800A6EB0 000A3CB0  38 C3 2B 38 */	addi r6, r3, lbl_802F2B38@l
+/* 800A6EB0 000A3CB0  38 C3 2B 38 */	addi r6, r3, volume@l
 /* 800A6EB4 000A3CB4  38 66 00 0C */	addi r3, r6, 0xc
 /* 800A6EB8 000A3CB8  38 A0 00 08 */	li r5, 8
-/* 800A6EBC 000A3CBC  C0 22 93 C8 */	lfs f1, lbl_803CDD48-_SDA2_BASE_(r2)
+/* 800A6EBC 000A3CBC  C0 22 93 C8 */	lfs f1, unkFloat-_SDA2_BASE_(r2)
```

Comment:
Again, Please don't name labels something that you make up. Either name it what the symbol file calls it, or in this case, since there is no symbol for it (because it's a float), just leave it as `lbl_803CDD48` and use it in the CPP file as that label.